ext/hash/tests/hash-clone.phpt

```diff
@@ -24,6 +24,10 @@ foreach ($algos as $algo) {
 
     hash_update($copy, "Can’t tell if this is true or dream");
     var_dump(hash_final($copy));
+
+    if ($algo === "joaat") {
```

Comment:
Why the special carve out for joaat?

Reply:
I found some bugs in joaat. Next comment will explain more.